### DISCUSSION TITLE
How to exclude .git when diffing two directories?

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [dandavison/delta#1612] [Q&A]
Title.

**Comment by dandavison** [comment]
Hi @limaceous-bushwhacker, can you give an example of the command that you have been using to diff directories?

**Reply by limaceous-bushwhacker** [reply]
I just used `delta dir1 dir2`.

**Reply by dandavison** [reply]
Hm, interesting question. So firstly, let me say that `delta dir1 dir2` is just a shortcut for `git diff --no-index dir1 dir2 | delta`. I haven't added CLI options to control it -- my thinking is that when we need control, we should use the explicit form. (Note that this `git diff` command can be applied to any files or directories; they don't have to be in the same git repo).

However, I do not see a way to exclude certain paths when using `git diff` on two directories? Do you?

But I think what we can do is use traditional `diff -r`, being sure to pass `-u` for unified diff format:

```
diff -ru --exclude=.git dir1 dir2 | delta
```

How's that?
